site/docs/patterns/app-header.mdx

```diff
@@ -0,0 +1,178 @@
+    alt="Header paired with vertical navigation."
+    border
+  />
+
+## Behaviors
+
+### Responsive Behaviors
+
+The app header is responsive to the width of the application viewport and responds dynamically to the [Salt breakpoints](/salt/foundations/responsiveness#breakpoints). As the app header scales down to fit smaller viewports, its elements adjust dynamically to provide the best possible user experience. 
+
+For small viewport experiences, the app logo remains visible, while navigation and utility actions collapse into a hamburger menu positioned to left of the brand logo. The navigation items will sit on top of the utility actions in the hierarchy of the vertical navigation. 
+
+#### Desktop
+
+  <Diagram
+    src="/img/patterns/app-header/responsive-desktop.png"
+    alt="App header with desktop layout for medium, large and extra large viewport experiences."
+    border
+  />
+
+#### Mobile
+
+  <Diagram
+    src="/img/patterns/app-header/responsive-mobile.png"
+    alt="App header with mobile layout for extra small and small viewport experiences."
+    border
+  />
+
+<ExampleContainer type="positive">
+ - Navigation items sit on top of the utility actions in the hierarchy of the hamburger menu.
+ - If search utility action is part of the app header, keep search functionality persistently available for small viewports. 
+ - Consider medium, large and extra large [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) for desktop experiences. 
+ - Consider extra small and small [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) for mobile experiences.
+</ExampleContainer>
+
+### Sticky header position
+
+The app header should sit in the north region, above the main, in a sticky position while the user scrolls down the page. This increases the discoverability of the elements in the header, where users can quickly access the navigation and utility actions elements without scrolling up to the top of the page. 
+
+  <Diagram
+    src="/img/patterns/app-header/sticky-header.png"
+    alt="The header stays in place while user scrolls."
+    border
+  />
+
+## Accessibility
+
+### Using skip links to bypass the app header
+
+If you're implementing a complex app header with several navigation items and utility actions, you should precede the component with a skip link. This will enable keyboard and screen reader users to quickly shift focus to the main page content, and navigate your application more efficiently.
+
+You can display and set focus to a skip link when the user moves focus from the previous component in the tab sequence to the app header—and configure the skip link to move focus to the main page content or an alternative target.
+
+  <Diagram
+    src="/img/patterns/app-header/skip-link.png"
+    alt="The 'skip to content' link should be the first focusable element in the header."
+    border
+  />
+
+*The "skip to content" link should be the first focusable element in the header.*
```

Comment:
I'm not sure what this means? At the beginning of this section is says the skip link should precede it but then it talk about it being in the header